src/utils/Operator.utils.js

```diff
@@ -103,7 +103,8 @@ export const successOperatorMap = (
   parameterName
 ) => {
   return operatorParameters.map((parameter) => {
-    const validParameterValue = parameterValue || undefined;
+    const validParameterValue =
+      parameterValue || parameterValue == 0 ? parameterValue : undefined;
```

Comment:
O `==` aqui é proposital? Se for tudo bem, mas acho legal sempre tomar cuidado com ele porque ele pode dar uns resultados inesperados. Ex:

![image](https://user-images.githubusercontent.com/36794274/127656695-a9684877-06ec-4ff1-9a7a-3a21b848c27c.png)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Equality